bentoml/tracing/_zipkin.py

```diff
@@ -12,6 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# This module name starts with underscore because it contains optional Python package
```

Comment:
@ssheng looks like walk_packages will skip modules starting with underscore, so I used it for these two modules that has optional dependency and we want to avoid loading if it's not used

Reply:
Correct and this is probably fine for now. The long term solution is still limit `wire` to BentoML modules.